notebooks/spark-intro.ipynb

```diff
@@ -0,0 +1,1026 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "95ca3df2-cad2-4e16-8768-bfca026bb576",
+   "metadata": {},
+   "source": [
+    "# Spark RDDs\n",
+    "This notebook is intended to teach you the basic structure of Apache Spark, from the basic structures to some of the more advanced techniques.  We'll do our best to give you practical advice, and not get bogged down in little minutia."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0e323ed5-86bc-48aa-a85d-4c53d546d1d0",
+   "metadata": {},
+   "source": [
+    "## Initializing Spark\n",
+    "We must begin by making sure that the needed JARs are present on the system, and that the interpreter knows we're loading them into the classpath."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "c88193e7-7e15-4c0e-acc9-e0ab458297ec",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "\u001b[32mimport \u001b[39m\u001b[36m$ivy.$\u001b[39m\n",
+       "\u001b[32mimport \u001b[39m\u001b[36m$ivy.$\u001b[39m\n",
+       "\u001b[32mimport \u001b[39m\u001b[36m$ivy.$\u001b[39m"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import $ivy.`org.apache.logging.log4j:log4j-core:2.17.0`\n",
+    "import $ivy.`org.apache.logging.log4j:log4j-1.2-api:2.17.0`\n",
+    "import $ivy.`org.apache.spark::spark-sql:3.3.1`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1a8b1bc1-e27a-489f-97c0-a2884978f841",
+   "metadata": {},
+   "source": [
+    "As a convenience, let's also quiet the logging facility."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "c2fcb3ca-1ce2-41e4-96e8-2e091a18cf24",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "\u001b[32mimport \u001b[39m\u001b[36morg.apache.log4j.{Level, Logger}\u001b[39m"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import org.apache.log4j.{Level, Logger}\n",
+    "Logger.getRootLogger.setLevel(Level.WARN)\n",
+    "Logger.getLogger(\"org\").setLevel(Level.OFF)\n",
+    "Logger.getLogger(\"org\").setLevel(Level.OFF)"
```

Comment:
Could add a comment to explain that silencing 'org' silences all `org.etc.etc` logging? Might be useful for some